spec/fixtures/iex/cash_flow/msft.yml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
I did the same thing for these yml files as I mentioned here: https://github.com/dblock/iex-ruby-client/pull/91#discussion_r517024181

Will do the same here based on the outcome of that conversation.

Reply:
Yeah I definitely don't want tests that are based on docs and not on actual interactions with IEX. Thanks.

Reply:
If you don't figure it out (get a paid account), i can try to take over the PR and try to record interactions with a real key

Reply:
I was going to pay for the account after getting this spun up for my own project so i guess it can't hurt to pay for it now and do this the right way. Will update tomorrow or on the weekend.

Reply:
Updated with proper fixtures.